samples/bot-formatting-cards/csharp/BotFormattingCards/Resources/mentionSupport.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.0",
+  "type": "AdaptiveCard",
+  "body": [
+    {
+      "type": "TextBlock",
+      "weight": "bolder",
+      "size": "large",
+      "text": "Mention Support Card"
+    },
+    {
+      "type": "TextBlock",
+      "text": "Hi <at>Adele UPN</at>, <at>Adele Azure AD</at>"
+    }
+  ],
+  "msteams": {
+    "entities": [
+      {
+        "type": "mention",
+        "text": "<at>Adele UPN</at>",
+        "mentioned": {
+          "id": "AdeleV@contoso.onmicrosoft.com",
+          "name": "Adele Vance"
+        }
+      },
```

Comment:
Not sure if we can hardcode this user.
Could you please add a note in readme file to update this json with relevant data ?

Reply:
Fixed